DEVELOPING.md

```diff
@@ -16,11 +16,27 @@ The resulting image is called openjdk
 ### Cloud build
 To build using the [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/overview), you need to have the [Google Cloud SDK](https://cloud.google.com/sdk/) installed locally.
 ```
-$ ./scripts/cloudbuild.sh
+$ PROJECT_ID=my-project
+$ ./scripts/cloudbuild.sh gcr.io/$PROJECT_ID/openjdk
 ```
 
 If you would like to simulate the cloud build locally, pass in the `--local` argument.
 ```
 $ PROJECT_ID=my-project
-$ ./scripts/cloudbuild.sh --local
-```
\ No newline at end of file
+$ ./scripts/cloudbuild.sh gcr.io/$PROJECT_ID/openjdk --local
+```
+
+# Running Tests
+Integration tests can be run via [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/overview).
+These tests deploy a sample test application to App Engine using the provided runtime image, and 
+exercise various integrations with other GCP services.
+```bash
+$ PROJECT_ID=my-project
+$ ./scripts/integration_test.sh gcr.io/$PROJECT_ID/openjdk
```

Comment:
This normally will not work, even after doing a cloud build first because the `:latest` tag will not be available. Maybe you should say, first find the image you want to test against by using
```
gcloud beta container images list-tags gcr.io/$PROJECT_ID/openjdk
```